assemble/bin/accumulo-cluster

```diff
@@ -210,7 +210,7 @@ function start_all() {
 
   for queue in $COMPACTION_QUEUES; do
     Q="COMPACTOR_HOSTS_${queue}"
-    for compactor in "${!Q}"; do
+    for compactor in ${!Q}; do
```

Comment:
I changed the compactor stuff too because I wanted to test it while I had my cluster up.